src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000044> (seawater freezing process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.waterencyclopedia.com/Re-St/Sea-Water-Freezing-of.html") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000044> "A material transformation process during which seawater freezes into sea ice.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000044> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000044> "seawater freezing process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00002200>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000045> (brine rejection process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Brine_rejection") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000045> "A material transformation process during which brine is formed due to the expulsion of salt during frazil formation or a seawater freezing process. Expelled salt accumulates in brine droplets, which remain in a liquid state due to their increased salinity.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000045> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000045> "brine rejection process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000046> (frazil)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000046> "Small needle-like ice crystals consisting of nearly pure fresh water, which are suspended in water.")
```

Comment:
@pbuttigieg I'll update the label to frazil ice,

Update to your definition:
> Ice which is composed of small, needle-like crystals consisting of nearly pure fresh water, which are suspended in water

I'll add the subclass axiom:
> 'surrounded by' some 'sea water'

editors note:
> Needs to have axioms added pertaining to the semantics of colloids and suspensions once they have been resolved see https://github.com/pato-ontology/pato/issues/91#issuecomment-290711238